projects/kit/components/input-chip/input-chip.directive.ts

```diff
@@ -118,13 +118,17 @@ export class TuiInputChipBaseDirective<T>
 
     protected onBackspace(key: string): void {
         // (keydown.backspace) doesn't emit event on empty input in ios safari
-        if (
-            key === 'Backspace' &&
-            !this.textfield.value() &&
-            this.interactive() &&
-            (this.mobile || !this.textfield.item)
-        ) {
-            this.onChange(this.value().slice(0, -1));
+        if (key === 'Backspace' && !this.textfield.value() && this.interactive()) {
+            if (this.mobile || !this.textfield.item) {
+                this.onChange(this.value().slice(0, -1));
+            } else {
+                this.textfield.input?.nativeElement?.dispatchEvent(
+                    new KeyboardEvent('keydown', {
+                        key: 'ArrowLeft',
+                        bubbles: true,
+                    } satisfies KeyboardEventInit),
```

Comment:
`satisfies` seems meaningless here, it's already type safe

Reply:
fixed